pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/test/ByteBufferSerDe.java

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Fix the license header - "removing the streamlio copyright line"

Reply:
fixed all headers